### PR TITLE
merge block timestamp test

### PR DESCRIPTION
**Pull request by MarcusWentz**
-Genesis block timestamp starts at 950
-All other timestamps increase by 1000 per block

### CODE REVIEW

File: src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/timestampPerBlockFiller.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      nonce: 1
+      code: ''
+      storage: {}
+
+    #Store the timestamp for each block produced.
+    0xcccccccccccccccccccccccccccccccccccccccc:
+      balance: 0x10000000000
+      nonce: 1
+      code: |
+         :yul { sstore(sub(number(),1), timestamp() ) }
```
Comment (winsvega) [inline src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/timestampPerBlockFiller.yml]: the timestamp return current block timestamp. why -1  ?

File: src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/timestampPerBlockFiller.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      accessList: []
+      gasLimit: 100000
+      maxPriorityFeePerGas: 0
+      maxFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+  # Block 2
+  - transactions:
```
Comment (winsvega) [inline src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/timestampPerBlockFiller.yml]: make a "blockHeader" overwrite here, set the timestamp diff  "RelTimestamp" keyword. 
note looks like we never tried a negative timestamp. if it does not work, use timestamp fields overwrite for  negative timestamp change.

Comment (winsvega) [reply src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/timestampPerBlockFiller.yml]: the cases 
-1, 0, 1, 9, 1000

File: src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/timestampPerBlockFiller.yml (hunk 1 above)
Comment (MarcusWentz) [reply src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/timestampPerBlockFiller.yml]: Now set to the current block.

File: src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/timestampPerBlockFiller.yml (hunk 2 above)
Comment (MarcusWentz) [reply src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/timestampPerBlockFiller.yml]: Added RelTimestamp and timestamp block header tests.

**Comment by winsvega** [comment]
https://github.com/ethereum/tests/pull/1060

**Pull request closed without merging**